package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "handlebars",
   "barename": "handlebars",
-  "version": "1.3.3-sugarcrm-temporary",
+  "version": "1.3.4-sugarcrm-temporary",
```

Comment:
This is temporary for testing purposes. Actual version will be done after QA. Version update needs to be done here and in other places where it references to the old version